README.md

```diff
@@ -1,266 +1,166 @@
-Use the following commands to run a standalone MinIO server as a container.
-
-Standalone MinIO servers are best suited for early development and evaluation. Certain features such as versioning, object locking, and bucket replication
-require distributed deploying MinIO with Erasure Coding. For extended development and production, deploy MinIO with Erasure Coding enabled - specifically,
-with a *minimum* of 4 drives per MinIO server. See [MinIO Erasure Code Overview](https://min.io/docs/minio/linux/operations/concepts/erasure-coding.html)
-for more complete documentation.
-
-### Stable
-
-Run the following command to run the latest stable image of MinIO as a container using an ephemeral data volume:
+MinIO generally supports deployment on 64-bit Linux or Kubernetes-based infrastructure.
+The fastest way to start experimenting with MinIO is to run it as a container on local machine.
+The following command uses [podman](https://podman.io/) and binds a local volume ``~/minio`` to the container for storage:
 
 ```sh
-podman run -p 9000:9000 -p 9001:9001 \
-  quay.io/minio/minio server /data --console-address ":9001"
+podman run \
+  -p 9000:9000 \
+  -p 9090:9090 \
+  -v ~/minio:/data \
+  -n minio \
+  quay.io/minio/minio server /data --console-address ":9090"
 ```
 
-The MinIO deployment starts using default root credentials `minioadmin:minioadmin`. You can test the deployment using the MinIO Console, an embedded
-object browser built into MinIO Server. Point a web browser running on the host machine to <http://127.0.0.1:9000> and log in with the
-root credentials. You can use the Browser to create buckets, upload objects, and browse the contents of the MinIO server.
+MinIO starts using the default root credentials ``minioadmin:minioadmin`` and outputs it's logs to the terminal or shell.
+You can pass the ``-dt`` parameters to run the MinIO container in a detached state with TTY.
 
-You can also connect using any S3-compatible tool, such as the MinIO Client `mc` commandline tool. See
-[Test using MinIO Client `mc`](#test-using-minio-client-mc) for more information on using the `mc` commandline tool. For application developers,
-see <https://min.io/docs/minio/linux/developers/minio-drivers.html> to view MinIO SDKs for supported languages.
+Once started, you can perform object storage operations by pointing any [S3-compatible SDK](https://min.io/docs/minio/linux/developers/minio-drivers.html?ref=github-minio) to ``localhost:9000``.
+Alternatively, open ``localhost:9090`` in your preferred browser to use the [MinIO Console](https://min.io/docs/minio/linux/administration/minio-console.html?ref=github-minio).
 
-> NOTE: To deploy MinIO on with persistent storage, you must map local persistent directories from the host OS to the container using the `podman -v` option. For example, `-v /mnt/data:/data` maps the host OS drive at `/mnt/data` to `/data` on the container.
+For more complete documentation on installing and deploying MinIO, see the following:
```

Comment:
I'd add an h2 just above this:

> `## Official Documentation`

> `### Production Deployments'

   Links to prod quickstarts

> `### Local Testing Deployments`

   Links to OS quickstarts.

Reply:
Might also consider moving all of these links to the bottom with the Explore Further links.
Though I think we need _some_ link to docs site(s) close to the top of this.